spec/models/user_search_spec.rb

```diff
@@ -12,6 +12,7 @@
   let(:user4)     { Fabricate :user, username: "mrpink",   name: "Steve Buscemi",  last_seen_at: 7.days.ago }
   let(:user5)     { Fabricate :user, username: "mrbrown",  name: "Quentin Tarantino", last_seen_at: 6.days.ago }
   let(:user6)     { Fabricate :user, username: "mrwhite",  name: "Harvey Keitel",  last_seen_at: 5.days.ago }
+  let(:user7)     { Fabricate :user, name: "Régis" }
```

Comment:
Yes. From what I've experienced, debouncing for autocomplete seems bumped. But it's another issue.